UnityClient/Assets/Scripts/TowerDefence/Enemy.cs

```diff
@@ -5,10 +5,14 @@
 
 public class Enemy : MonoBehaviour
```

Comment:
It seems this entire type is breaking our conventions. I've pointed out what I can but some of the stuff is not part of this branch so I can't point it out. Please apply things that seem to apply in places I've not marked out. Thanks.